packages/components/cascader-panel/src/node.vue

```diff
@@ -28,7 +28,7 @@
       @update:model-value="handleSelectCheck"
     />
     <el-radio
-      v-else-if="checkStrictly"
+      v-else-if="checkStrictly && !panel.checkOnClickNode"
```

Comment:
The tree component keep the checkbox when `checkOnClickNode` is true, i think it's better to reflect the behavior for Cascader too.
```suggestion
      v-else-if="checkStrictly"
```
But maybe we need some other opinion.
WDYT ? cc @warmthsea @btea @kooriookami

Reply:
@snowbitx Well done, it's great.

In the future we'll would like to have the same prop `showRadio` for the `multiple` mode, 
we can have ~~showRadio~~ → **showPrefix** for both usage.

Reply:
> @snowbitx Well done, it's great.
> 
> In the future we'll would like to have the same prop `showRadio` for the `multiple` mode, we can have ~showRadio~ → **showPrefix** for both usage.

Got it — updated to use showPrefix  in the latest commit ✅